src/core/csv/reader.cc

```diff
@@ -656,79 +663,51 @@ const char* GenericReader::repr_binary(
 // Input handling
 //------------------------------------------------------------------------------
 
-void GenericReader::open_input() {
-  if (input_mbuf) return;
-  double t0 = wallclock();
-  CString text;
-  const char* filename = nullptr;
-  if (fileno > 0) {
-    #if DT_OS_WINDOWS
-      throw NotImplError() << "Reading from file-like objects, that involves "
-        << "file descriptors, is not supported on Windows";
-    #else
-      const char* src = src_arg.to_cstring().ch;
-      input_mbuf = Buffer::mmap(src, 0, fileno, false);
-      size_t sz = input_mbuf.size();
-      trace("Using file %s opened at fd=%d; size = %zu", src, fileno, sz);
-    #endif
-  } else if ((text = text_arg.to_cstring())) {
-    size_t size = static_cast<size_t>(text.size);
-    input_mbuf = Buffer::external(text.ch, size + 1);
-    input_is_string = true;
-
-  } else if ((filename = file_arg.to_cstring().ch) != nullptr) {
-    input_mbuf = Buffer::mmap(filename);
-    size_t sz = input_mbuf.size();
-    trace("File \"%s\" opened, size: %zu", filename, sz);
-
-  } else {
-    throw IOError() << "No input given to the GenericReader";
-  }
-  line = 1;
-  sof = static_cast<const char*>(input_mbuf.rptr());
-  eof = sof + input_mbuf.size();
-  if (verbose) {
-    trace("==== file sample ====");
-    const char* ch = sof;
-    bool newline = true;
-    for (int i = 0; i < 5 && ch < eof; i++) {
-      if (newline) trace("%s", repr_source(ch, 100));
-      else         trace("...%s", repr_source(ch, 97));
-      const char* start = ch;
-      const char* end = std::min(eof, ch + 10000);
-      while (ch < end) {
-        char c = *ch++;
-        // simplified newline sequence. TODO: replace with `skip_eol()`
-        if (c == '\n' || c == '\r') {
-          if ((*ch == '\r' || *ch == '\n') && *ch != c) ch++;
-          break;
-        }
-      }
-      if (ch == end && ch < eof) {
-        ch = start + 100;
-        newline = false;
-      } else {
-        newline = true;
-      }
-    }
-    trace("=====================");
-  }
-  t_open_input = wallclock() - t0;
```

Comment:
I don't see this line moved anywhere, don't we measure `t_open_input` anymore?

Reply:
yep, added it back